src/kOS/Communication/kOSConnectivityParameters.cs

```diff
@@ -64,6 +64,8 @@ public override bool HasPresets
             }
         }
 
+        public override string DisplaySection => "kOS";
+
```

Comment:
I have to stick with the version of Monodevelop that ships with the version of Unity that KSP is running on, or else I can't get the debug version to work.